### DISCUSSION TITLE
Bluetooth doesn't connect with Miao

### DISCUSSION THREAD

**Discussion by maram403** [NightscoutFoundation/xDrip#3682] [Q&A]
Android 6, the X-Drip update, the latest version of Bluetooth, does not communicate with the Miao![image](https://github.com/user-attachments/assets/8fac83ae-e739-400d-b17b-bba31e076bfd)

**Comment by Navid200** [comment]
Unfortunately, we have had to retire Android 6.

xDrip is now only supported for Android 7 and newer.

**Reply by maram403** [reply]
There is no way to download an old version

**Reply by Navid200** [reply]
It's not a good idea to use an old version.  Nevertheless, you can download old versions as well as new versions from the repository.
You can also see them as well as corresponding release notes here: https://navid200.github.io/xDrip/docs/ReleaseNotes.html
